python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -302,17 +312,23 @@ def runAssociation(self, sensorRef):
         catalog = sensorRef.get(diffType + "Diff_diaSrc")
         diffim = sensorRef.get(diffType + "Diff_differenceExp")
 
-        dia_sources = self.diaSourceDpddifier.run(catalog,
-                                                  diffim,
-                                                  return_pandas=True)
+        diaSources = self.diaSourceDpddifier.run(catalog,
+                                                 diffim,
+                                                 return_pandas=True)
+        # Load the DiaObjects and DiaSource history.
+        loaderResult = self.diaCatalogLoader.run(diffim, self.apdb)
```

Comment:
I guess this line is outside the `try` block because if loading fails, then something's wrong with the DB and we can't do anything with it anyway? Might be worth a comment.